### DISCUSSION TITLE
Hardware Compatibility List

### DISCUSSION THREAD

**Discussion by gitoutsub** [virt-manager/virt-manager#688] [Q&A]
Hello,
 Is there a list of compatible hardware that plays well with virtualization in general and qemu/kvm/virt-manager specifically? I have an Nvidia GTX 1650 (TU117) and can not add that hardware in the VM details. When I try to add, it just sits and spins and never completes the creation of the domain. Using the latest drivers for that card and the BIOS is up to date. Or is there another setting that I'm missing that prevents its addition?
 
Thanks

**Comment by phrdina** [comment]
Hi, sorry for taking long time to reply, there is no such list. You are most likely talking about attaching GPU to your VM, search for `virt-manager gpu assignment`, `virt-manager gpu passthrough` as there is a lot of guides and tutorials.
